ArticleTemplates/assets/js/modules/more-tags.js

```diff
@@ -14,7 +14,7 @@ define(function () {
             initialised = true;
             tags = document.querySelectorAll('.tags .inline-list .inline-list__item');
 
-            if (tags.length > SHOW_TAGS + 1) {
+            if (tags.length > SHOW_TAGS) {
                 moreButton = document.createElement('li');
                 moreButton.id = 'more-tags-container';
                 moreButton.classList.add('inline-list__item');
```

Comment:
I think you'll need to add the new class name 'tags__list-item' to moreButton so it picks up the styles.